### DISCUSSION TITLE
Photoprism password

### DISCUSSION THREAD

**Discussion by coffeecupdad** [tteck/Proxmox#273] [General]
I have installed using the Photoprism script, and at the console the login fails. I am using admin/admin per docs:
![image](https://user-images.githubusercontent.com/53222542/172652154-94409791-83ee-4ff6-868e-7ec3aef0d102.png)
Any advice?

**Comment by tteck** [comment]
That looks like you're at the root prompt, so `root` and whatever password you entered at creation.

**Comment by coffeecupdad** [comment]
Thank you. I ended up resetting the PW.
https://dannyda.com/2020/12/23/how-to-reset-forgotten-lxc-container-root-password-on-proxmox-vepve/

**Reply by tteck** [reply]
I use automatic login 😉
